fullgame/bosses-glitchless.md

```diff
@@ -17,24 +17,26 @@
 	- The Elder Guardian
 	- The Warden
 
-## All Bosses (Pre 1.19)
+## All Bosses (Pre-1.19)
 * Must kill the following bosses:
 	- The Ender Dragon
 	- The Wither
 	- The Elder Guardian
 
-* You do not have to use the versions named in the version splits, they are only correlating to which bosses you must kill.
+* You do not have to use the versions named in the version splits, they are
+only correlating to which bosses you must kill.
 
 ## Wither
 
-* The same rules apply as All Bosses, however you are only required to kill
+* The same rules apply as for All Bosses, however you are only required to kill
 the Wither.
 
 ## Elder Guardian
 
-* The same rules apply as All Bosses, however you are only required to kill
+* The same rules apply as for All Bosses, however you are only required to kill
 the Elder Guardian.
 
 ## Warden
 
-* The same rules apply as All Bosses, however you are only required to kill the Warden
+* The same rules apply as for All Bosses, however you are only required to kill
+the Warden.
```

Comment:
I am not sure how to feel about this, but "the warden" is lowercased on minecraft.net (as opposed to "The End" or "the Wither")

https://www.minecraft.net/en-us/article/meet-warden
```suggestion
the warden.
```

Reply:
I agree with this.